bolt-design-system/bolt
# Feature/nightwatch testing experiments

## Summary

Update Nightwatch to 1.0, reconfigure to have Travis properly run tests on Sauce Labs in both Chrome and IE11.

## Details

Nightwatch was updated to v1.0. Tests are now running again in both Chrome and IE11 through Sauce labs.

## How to test

See the Travis build log for testing results. e2e testing lines start with an obvious text highlighted block. They should be prefixed with either 'chrome' or 'ie11' and include the name of the test that is running.
